cylc/flow/task_events_mgr.py

```diff
@@ -643,12 +651,16 @@ def process_message(
             submit_num:
                 The submit number of the task relevant for the message.
                 If not specified, use latest submit number.
+            forced:
+                If this message is due to manual completion or not (cylc set)
 
         Return:
             None: in normal circumstances.
             True: if polling is required to confirm a reversal of status.
 
         """
+        # TODO - clear up confusing use of TASK_OUTPUTs vs TASK_EVENTs as
+        #        task messages below.
```

Comment:
TODO

Reply:
This is something I noticed while working on this branch, but fixing it is off-topic ... I've removed the comment and put up a new issue: #6002